snap/snapcraft.yaml

```diff
@@ -60,32 +60,32 @@ parts:
       go mod tidy
       make build
 
-      install -DT "./cmd/device-rest-go" "$SNAPCRAFT_PART_INSTALL/bin/device-rest-go"
+      install -DT "./cmd/device-rest" "$SNAPCRAFT_PART_INSTALL/bin/device-rest"
 
-      install -d "$SNAPCRAFT_PART_INSTALL/config/device-rest-go/res/"
+      install -d "$SNAPCRAFT_PART_INSTALL/config/device-rest/res/"
 
       cat "./cmd/res/configuration.toml" | \
-        sed -e s:"'./device-rest-go.log'":"'\$SNAP_COMMON/device-rest-go.log'": \
-        -e s:"ProfilesDir = './res/profiles'":"ProfilesDir = '\$SNAP_DATA/config/device-rest-go/res/profiles'":  \
-        -e s:"DevicesDir = './res/devices'":"DevicesDir = '\$SNAP_DATA/config/device-rest-go/res/devices'": > \
-        "$SNAPCRAFT_PART_INSTALL/config/device-rest-go/res/configuration.toml"
+        sed -e s:"'./device-rest.log'":"'\$SNAP_COMMON/device-rest.log'": \
+        -e s:"ProfilesDir = './res/profiles'":"ProfilesDir = '\$SNAP_DATA/config/device-rest/res/profiles'":  \
+        -e s:"DevicesDir = './res/devices'":"DevicesDir = '\$SNAP_DATA/config/device-rest/res/devices'": > \
+        "$SNAPCRAFT_PART_INSTALL/config/device-rest/res/configuration.toml"
```

Comment:
As with the last review, I think this can all just be reduced to a copy, as:

 * log files aren't used anymore
 * both Dir settings can be overridden as env overrides in the app definition

Reply:
Thanks, I'll update that and force-push

Reply:
@tonyespy - I've pushed a new commit to this PR, replacing the sed with an install statement - and also updating the environment overrides.